app/backend/app.py

```diff
@@ -76,10 +77,15 @@ def static_file(path):
 @app.route("/content/<path>")
 def content_file(path):
     blob = blob_container.get_blob_client(path).download_blob()
+    if not blob.properties or "content_settings" not in blob.properties:
```

Comment:
Thanks for the report! That error is actually due to an error in the Python SDK, which I've sent a PR for: https://github.com/Azure/azure-sdk-for-python/pull/30846

But you're still seeing a more fundamental issue, as the error indicates it either hasn't found the blob or is having an issue loading it. I'm guessing it's due to the characters in the filename, as I know we've had a few reports about non-ASCII characters, like:

https://github.com/Azure-Samples/azure-search-openai-demo/issues/334

Were you able to upload that file successfully? Is it in your search index?

Reply:
Thanks for sending PR in Python SDK.

> Were you able to upload that file successfully? Is it in your search index?
Yes, I am able to upload the files successfully. The problem is only with filenames in Japanese in my case.
Also, content is present in my search index, and I am able to view the file index after the change I have suggested. 
Attached screenshot for reference:
![image](https://github.com/Azure-Samples/azure-search-openai-demo/assets/3011311/2352c11a-c977-4493-aa49-7d6a82951145)

Reply:
Okay, I'm glad they uploaded correctly. I'll make the change you suggested, since my PR for the Azure Python SDK will take some time to get merged and released. Thanks for the suggestion!

Reply:
Thank you for raising PR for the fix.